package.json

```diff
@@ -12,7 +12,7 @@
     "build:redirects": "cp ./_redirects ./website/build/babel",
     "build:repl": "cross-env NODE_ENV=production webpack",
     "build:codemirror6": "rollup -c rollup.config.mjs",
-    "lint": "eslint js website",
+    "lint": "eslint js website scripts",
```

Comment:
`scripts` are now linted as well. Most changes in `scripts/download-sponsors.js` are style changes.